java/query/src/test/java/sleeper/query/lambda/SqsQueryProcessorLambdaIT.java

```diff
@@ -223,7 +226,7 @@ public void shouldSplitUpQueryWhenItSpansMultiplePartitions() throws Exception {
                         builder.lastKnownState(QUEUED).recordCount(0L).build(),
                         builder.lastKnownState(QUEUED).recordCount(0L).build(),
                         builder.lastKnownState(QUEUED).recordCount(0L).build(),
-                        builder.lastKnownState(QUEUED).recordCount(0L).build());
+                        builder.lastKnownState(COMPLETED).recordCount(365L).build());
```

Comment:
From a glance at this test I wasn't sure why one of the queries was completed here, but that's because `processQuery` and `processQueriesFromQueue` processes both Query and LeafPartitionQuery queues.

Maybe we could have `processQuery` and `processLeafPartitionQuery` so we can see when we process each queue in the test?

Reply:
I believe this has now been addressed

Reply:
Nice, that reads much better, thanks!